lddtree.sh

```diff
@@ -25,6 +25,7 @@ usage() {
 	  -R <root>       Use this ROOT filesystem tree
 	  --no-auto-root  Do not automatically prefix input ELFs with ROOT
 	  --no-recursive  Do not recursivly parse dependencies
+	  --no-header     Do not show header (binary and interpreter info)
```

Comment:
So we can parse output and focus only on linked lib
usage example :
output=$(/lddtree/lddtree.sh --no-recursive --no-header $_path)